boards.txt

```diff
@@ -6904,6 +6904,266 @@ sparkfun_esp32s2_thing_plus.menu.EraseFlash.none.upload.erase_cmd=
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.noota_3gffat.upload.maximum_size=1048576
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.huge_app=Huge APP (3MB No OTA/1MB SPIFFS)
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.huge_app.build.partitions=huge_app
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FATFS)
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.fatflash.build.partitions=ffat
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9.9MB FATFS)
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.rainmaker=RainMaker 4MB
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.rainmaker.build.partitions=rainmaker
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.rainmaker.upload.maximum_size=1966080
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.rainmaker_4MB=RainMaker 4MB No OTA
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.rainmaker_4MB.build.partitions=rainmaker_4MB_no_ota
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.rainmaker_4MB.upload.maximum_size=4038656
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.rainmaker_8MB=RainMaker 8MB
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.rainmaker_8MB.build.partitions=rainmaker_8MB
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.rainmaker_8MB.upload.maximum_size=4116480
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.app5M_fat24M_32MB=32M Flash (4.8MB APP/22MB FATFS)
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.app5M_fat24M_32MB.build.partitions=large_fat_32MB
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.app5M_fat24M_32MB.upload.maximum_size=4718592
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.app5M_little24M_32MB=32M Flash (4.8MB APP/22MB LittleFS)
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.app5M_little24M_32MB.build.partitions=large_littlefs_32MB
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.app5M_little24M_32MB.upload.maximum_size=4718592
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.esp_sr_16=ESP SR 16M (3MB APP/7MB SPIFFS/2.9MB MODEL)
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.esp_sr_16.upload.maximum_size=3145728
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.esp_sr_16.upload.extra_flags=0xD10000 {build.path}/srmodels.bin
+sparkfun_esp32s3_thing_plus.menu.PartitionScheme.esp_sr_16.build.partitions=esp_sr_16
```

Comment:
Same here.

```suggestion
```

Reply:
made changes in last commit. Let me know if I should have created a branch before doing the pull request.

Reply:
No it's fine like it is :) we will merge soon. Thanks for the fixes.

Reply:
But yeah, maybe next time create a branch with proper naming for the derived "changes" ;)